proof/invariant-abstract/LevityCatch_AI.thy

```diff
@@ -19,9 +19,13 @@ arch_requalify_consts
 arch_requalify_facts
   ptrFormPAddr_addFromPPtr
   aobj_ref_arch_cap
+  arch_finalise_cap_bcorres
+  prepare_thread_delete_bcorres
 
 lemmas aobj_ref_arch_cap_simps[simp] = aobj_ref_arch_cap
 
+lemmas [wp] = arch_finalise_cap_bcorres prepare_thread_delete_bcorres
```

Comment:
Out of curiosity, why were these added to `LevityCatch_AI`? Is it that there's nowhere better for them? I would have thought that we would like to minimise its content, with it mainly containing lemmas that could be moved elsewhere.

Reply:
Definitely a derp. Would this be acceptable instead?
```
           apply (simp add: select_ext_fa[simplified free_asid_select_def]
                            free_asid_select_def o_def returnOk_liftE[symmetric]
                       split del: if_split)
```

Reply:
Just to make sure, I guess this was for the conversation just below?

Reply:
Oh, right, yes. Sorry, I missed this one as well!

Reply:
I wanted to have these in a generic theory file, but couldn't find one that was in the import hierarchy in time for these to be useful. I'll take another look while I'm fixing X64, but I don't know if I'll find anyplace better. Advice welcome.